src/submission/models.py

```diff
@@ -1386,6 +1386,7 @@ class FrozenAuthor(models.Model):
 
     institution = models.CharField(max_length=1000)
     department = models.CharField(max_length=300, null=True, blank=True)
+    frozen_biography = models.TextField(null=True, blank=True, verbose_name=_('Frozen Biography'))
```

Comment:
One last change needed @joemull:
We need to update the `submissions.forms.EditFrozenAuthor` class and add the `frozen_biography` field so that it can be edited by regular means if needed.

I would also add a help_text to this field similar to what we have for frozen_orcid, explaining how account/frozen author biography fields play with each other.

Reply:
OK I've added these things